### ISSUE TITLE
`buildout` cannot create start-up scripts using current entry point configuration

### ISSUE THREAD

**Issue by wesleybl** [robotframework/robotframework#5098]
I can't run the code below:

```python
>>> import robot.run
>>> robot.run.run_cli
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'function' object has no attribute 'run_cli'
```

But I can run:

```python
>>> from robot.run import run_cli
>>> run_cli()
[ ERROR ] Expected at least 1 argument, got 0.

Try --help for usage information.
```

This is strange, because depending on the form of the import, `run` can be a function or a module.

**Comment by wesleybl** [comment]
This is related to #4129. The [buildout](https://github.com/buildout/buildout) generates the following code for the [entry point](https://github.com/robotframework/robotframework/blob/e592e2f441e59abb9eef34cbed878c8051d81edf/setup.py#L71):

```python
import robot.run

if __name__ == '__main__':
    sys.exit(robot.run.run_cli())
```

Then the above error occurs when running the script.

I don't think changing the way the entry point is written would solve the problem. I think what would solve it would be to eliminate the confusion of "run" in both types of import. Of course the buildout could be wrong too, but that doesn't invalidate this issue here.

cc @choeger

**Comment by wesleybl** [comment]
> but could you not switch the entry points to use the root module directly?

humm @choeger' suggestion works. If the entry point is changed to:

```python
'robot = robot:run_cli'
```

The script generated by buildout would be:

```python
import robot

if __name__ == '__main__':
    sys.exit(robot.run_cli())
```

and it would work.

**Comment by wesleybl** [comment]
@pekkaklarck I think it should be renamed because it causes confusion. But if it's going to cause a lot of trouble, could you at least change the entry point to work with buildout?

**Comment by wesleybl** [comment]
> Do you mean that in our setup.py we should change 'robot = robot.run:run_cli' to 'robot = robot:run_cli'?

Yes. Remembering that the other entrypoints will require the same change.

> That sounds fine, but I'm slightly worried about possible side-effects and believe it would be safer to do that in RF 7.1 than in 7.0.1.

The `run` module will continue to exist. So I don't see a problem with making this change in 7.0.1. I don't see anything breaking. I think it's a safe change to make.

But in any case, changing the module name cannot be lost sight of. Maybe in an 8.0 version?

**Comment by pekkaklarck** [comment]
I had forgotten about this. We are just about to release RF 7.3. I'll test does updating the entry point cause issues and will do that if it doesn't. Otherwise we need to think about module renaming and that would be RF 8 business.
